R/estimate_infections.R

```diff
@@ -22,7 +22,8 @@
 #'
 #' @param reported_cases Deprecated; use `data` instead.
 #'
-#' @param generation_time A call to [generation_time_opts()] defining the
+#' @param generation_time A call to [gt_opts()] or [generation_time_opts()]
```

Comment:
```suggestion
#' @param generation_time A call to [gt_opts()] (or its alias [generation_time_opts()])
```
trying to make clear here that they're the same